src/tool_cfgable.h

```diff
@@ -278,6 +278,7 @@ struct OperationConfig {
   long happy_eyeballs_timeout_ms; /* happy eyeballs timeout in milliseconds.
                                      0 is valid. default: CURL_HET_DEFAULT. */
   bool haproxy_protocol;          /* whether to send HAProxy protocol v1 */
+  char *haproxy_clientip;         /* spoofed client IP for HAProxy protocol */
```

Comment:
Please don't call it "spoofed". It *is* the client IP since the application says so

Reply:
Got it